src/css/doc.css

```diff
@@ -494,15 +494,16 @@ ul ul ul {
 }
 
 .doc .quoteblock {
-  margin-top: var(--base-medium-space);
+  margin-top: var(--base-extra-large-space);
 }
 
 .doc .abstract blockquote {
-  font-size: 0.9375rem;
+  /* font-size: 0.9375rem; */
```

Comment:
can just be deleted if we go ahead with this change?
e.g. we have version control, let's keep the existing CSS clean.

Reply:
Yeah, fair enough. I saw other commented lines in our CSS so I really wasn't sure what our approach was. (There are a lot of comments)